bin/change_log.ml

```diff
@@ -0,0 +1,47 @@
+(*---------------------------------------------------------------------------
+   Copyright (c) 2016 Daniel C. Bünzli. All rights reserved.
+   Distributed under the ISC license, see terms at the end of the file.
+   %%NAME%% %%VERSION%%
+  ---------------------------------------------------------------------------*)
+
+open Bos_setup
+open Dune_release
+
+let print_latest_change_log_entry () =
+  let pkg = Pkg.v ~dry_run:true () in
+  Pkg.change_log pkg >>= Text.change_log_file_last_entry
+  >>| (fun (version, (title, body)) ->
```

Comment:
I was about to point out that it would be nice to use the `let*` operators, but given all the other entrypoints are also using infix operators it's ok too. Up to you.